### ISSUE TITLE
Add `reduction` parameter to all loss functions and metrics.

### ISSUE THREAD

**Issue by zakajd** [photosynthesis-team/piq#92]
**Is your feature request related to a problem? Please describe.**
Some loss functions have this parametr, others don't.
In SSIM and MS-SSIM there is also `size_average` flag which is redundant and copies old PyTorch loss style. 

**Describe the solution you'd like**
Clear interfaces for SSIM, MS-SSIM
Add reduction to VIF and any other metric/loss that don't have it
